src/Tizen.NUI/src/public/BaseComponents/ViewBindableProperty.cs

```diff
@@ -2675,6 +2688,27 @@ private void SetBackgroundImageBorder(Rectangle value)
             Tizen.NUI.Object.SetProperty((System.Runtime.InteropServices.HandleRef)SwigCPtr, View.Property.BACKGROUND, new PropertyValue(map));
         }
 
+        private void SetBorderlineColor(Color value)
+        {
+            if (value == null)
+            {
+                return;
+            }
+
+            (backgroundExtraData ?? (backgroundExtraData = new BackgroundExtraData())).BorderlineColor = value;
+
+            PropertyMap map = Background;
+
+            if (map.Empty())
+            {
+                return;
+            }
+
+            map[Visual.Property.BorderlineColor] = new PropertyValue(value);
+
+            Tizen.NUI.Object.SetProperty((System.Runtime.InteropServices.HandleRef)SwigCPtr, View.Property.BACKGROUND, new PropertyValue(map));
```

Comment:
We'd better use virtual API `ApplyBorderline` instead of this thing.

`ApplyBorderline` automatically create `Transparent` background if nobody settup Background things.
and, `ImageView` has it's own borderline logic (due to the `Image` drawed on `ImageView.Property.IMAGE` level.)
And also, It did'nt newly create new Visual.

Reply:
Thank you for your comment!
I have replaced the code with ApplyBorderline();